.github/workflows/upstream_testing.yml

```diff
@@ -0,0 +1,13 @@
+---
+name: "Nautobot Upstream Testing"
+
+on:  # yamllint disable-line rule:truthy rule:comments
+  schedule:
+    - cron: "0 4 */2 * *"  # every other day at midnight
+
+jobs:
+  upstream-test:
+    uses: "nautobot/nautobot/.github/workflows/plugin_upstream_testing_base.yml@develop"
+    with:  # Below could potentially be collapsed into a single argument if a concrete relationship between both is enforced
+      invoke_context_name: "NAUTOBOT_SSOT_ARISTACV"
```

Comment:
Ensure this is the right invoke context hame, it appears different here:

https://github.com/nautobot/nautobot-plugin-ssot-arista-cloudvision/pull/150/files#diff-2e746c9eacad6e2c2fdefdc6a665b5ce4607ea384ebf97a729044b79acc778d6R36

Reply:
They look to match from what I can tell. Which invoke context name are you seeing?